src/pages/zh-cn/core-concepts/framework-components.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: 框架组件
+description: 学习如何使用 React，Svelte 等框架。
+---
+
+你可以在无需舍弃你所喜欢的组件框架的情况下使用 Astro 构建站点。
+
+Astro 支持多个受欢迎的框架，包括[React](https://reactjs.org/)、[Preact](https://preactjs.com/)、[Svelte](https://svelte.dev/)、[Vue](https://vuejs.org/)、[SolidJS](https://www.solidjs.com/)、[AlpineJS](https://alpinejs.dev/) 和 [Lit](https://lit.dev/)。
+
+## 安装集成
+
+Astro 有可供选择的 React、Preact、Svelte、Vue、SolidJS 和 Lit 集成。你可以可以在项目中安装和配置这些 Astro 集成中的一个或几个。
+
+
+要在 Astro 中使用这些框架，首先要安装该集成以及任何相关的对等依赖。
+
+```bash
+npm install --save-dev @astrojs/react react react-dom
+```
+
+然后在  `astro.config.mjs` 中导入并添加函数到集成列表中：
+
+```js
+import { defineConfig } from 'astro/config';
+
+import react from '@astrojs/react';
+import preact from '@astrojs/preact';
+import svelte from '@astrojs/svelte';
+import vue from '@astrojs/vue';
+import solid from '@astrojs/solid-js';
+import lit from '@astrojs/lit';
+
+export default defineConfig({
+	integrations: [react(), preact(), svelte(), vue(), solid() , lit()],
+});
+```
+
+⚙️ 阅读[集成指引](/zh-cn/guides/integrations-guide/)获取更多关于安装和配置 Astro 集成和信息。
+
+⚙️ 想要看看你选择的框架的示例？访问 [astro.new](https://astro.new/) 然后选择一个框架模板。
+
+## 使用框架组件
+
+在 Astro 页面、布局和组件中就像 Astro 组件一样使用你的 JavaScript 框架组件。所有组件都可放在 `/src/components` 目录中，或者你也可以放在任何你喜欢的地方。
+
+要使用框架组件，你需要在 Astro 组件脚本中使用相对路径（包括文件扩展名）导入它们。然后在其他组件、HTML 元素和类 JSX 表达式中使用它们。
+
+```astro
+---
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<html>
+  <body>
+    <h1>Use React components directly in Astro!</h1>
+    <MyReactComponent />
+  </body>
+</html>
+```
+
+:::tip
+记住：所有导入必须在 Astro 组件脚本的**顶部**。
+:::
+
+默认情况下，你的框架组件将渲染为静态 HTML。这对于模板组件而言非常有用，它不需要交互和避免分发没用的 JavaScript 给用户。
+
+## 激活组件
```

Comment:
Yes, because the menu was not translated by me, I only did the review.

I had planned to do a major cleanup and proofreading after all the documents were translated (including standardizing the wording, etc.)

Reply:
Awesome! Well feel free to just update that one menu item for now if you think a there’s an obviously clearer term. Thanks for all you work!